projects/cashmere/src/lib/button/button.component.spec.ts

```diff
@@ -55,15 +55,15 @@ describe('ButtonComponent', () => {
         }).compileComponents();
     }));
 
-    it('should apply class based on color property', () => {
+    it('should apply class based on buttonStyle property', () => {
         const testApp = new TestAppReference();
-        const buttonColors = ['primary', 'primary-alt', 'destructive', 'neutral', 'secondary', 'tertiary'];
+        const buttonStyles: ButtonStyle[] = ['primary', 'primary-alt', 'destructive', 'neutral', 'secondary', 'link', 'link-inline'];
 
-        buttonColors.forEach(color => {
-            testApp.setColor(color);
+        buttonStyles.forEach(style => {
```

Comment:
Because we are deprecated but not removed setColor we can't remove those tests. You'll need to test both methods to make sure they work. It's okay to change comments but wherever you see a setColor you'll need a setStyle also.

Reply:
fixed :+1: